2023/07.md

```diff
@@ -117,7 +117,7 @@ When applicable, use these emoji as a prefix to the agenda item topic.
     |   | 1     | 45m     | [Deferred Import Evaluation](https://github.com/tc39/proposal-defer-import-eval/) for Stage 2 ([slides](https://docs.google.com/presentation/d/1rSsVsFsnXQZ8pEGFwAGiVbVqndr4DHEUqTGEM9Au0_4), [spec](https://tc39.es/proposal-defer-import-eval/)) | Nicolò Ribaudo |
     |   | 1     | 60m     | Decimal: Open-ended discussion ([slides](https://docs.google.com/presentation/d/19MaO7On6knlweYZUei-d5VyqANzkKeR8QmC13KATvgs/)) | Jesse Alama |
     |   | 0     | 15m     | [DataView get/set Uint8C methods](https://github.com/ljharb/proposal-dataview-get-set-uint8c) for stage 1 or 2 or 3 ([spec](https://ljharb.github.io/proposal-dataview-get-set-uint8c/)) | Jordan Harband |
-    |   | 0     | 25m     | Optional chaining in assignment LHS for stage ? ([motivation](https://gist.github.com/nicolo-ribaudo/d264e424b618e7deaeca1d6e4f16a7c0), [spec](https://github.com/nicolo-ribaudo/ecma262/pull/4), [slides](https://docs.google.com/presentation/d/1KL9MRyxprgXDEsxT8Ddrdro074L3fQm88zXHsWL-Dwk)) | Nicolò Ribaudo |
+    |   | 0     | 25m     | [Optional chaining in assignment LHS](https://github.com/nicolo-ribaudo/proposal-optional-chaining-assignment) for stage ? ([spec](https://github.com/nicolo-ribaudo/ecma262/pull/4), [slides](https://docs.google.com/presentation/d/1KL9MRyxprgXDEsxT8Ddrdro074L3fQm88zXHsWL-Dwk)) | Nicolò Ribaudo |
```

Comment:
can you explicitly mention the stage possibilities?

Reply:
```suggestion
    |   | 0     | 25m     | [Optional chaining in assignment LHS](https://github.com/nicolo-ribaudo/proposal-optional-chaining-assignment) for stage 1/2 ([spec](https://github.com/nicolo-ribaudo/ecma262/pull/4), [slides](https://docs.google.com/presentation/d/1KL9MRyxprgXDEsxT8Ddrdro074L3fQm88zXHsWL-Dwk)) | Nicolò Ribaudo |
```